octoprint_displayprogress/__init__.py

```diff
@@ -52,17 +72,40 @@ def on_print_progress(self, storage, path, progress):
 	##~~ helpers
 
 	def _send_message(self, storage, path, progress):
+		symbol = self.__class__._get_symbol(self._settings)
+
 		message = self._settings.get(["message"]).format(progress=progress,
-		                                                 storage=storage,
-		                                                 path=path,
-		                                                 bar=self.__class__._progress_bar(progress))
+														 storage=storage,
```

Comment:
Use tabs for indenting and spaces for alignment :) Tabs up until the current indentiation level, then spaces to get where ever you want relative to the line:

```
[TAB]def.foo(bar,
[TAB]........fnord)
```

or in your example:

```
[TAB]def._send_message(self,.storage,.path,.progress):
[TAB][TAB]symbol.=.self.__class__._get_symbol(self._settings)

[TAB][TAB]message.=.self._settings.get(["message"]).format(progress=progress,
[TAB][TAB].................................................storage=storage,
[TAB][TAB].................................................path=path,
[TAB][TAB].................................................bar=self.__class__._progress_bar(progress,.symbol))
```

Always looks right regardless of selected tab width.